.circleci/lint_pr.sh

```diff
@@ -1,4 +1,4 @@
-set -xeo pipefail
+set -xe
```

Comment:
Verified in ssh session that script exists with non 0 exit code if there's lint failure.
![image](https://user-images.githubusercontent.com/5849952/217679071-020e6f17-f70e-4464-865c-d7aa7fcd14f3.png)

the pipefail was too aggressive for what we do below.